pages/trees/[id].tsx

```diff
@@ -208,39 +216,44 @@ const TreePage: TreePageWithLayout = ({ treeData, csrfToken }) => {
           >
             <Carousel dotsClass="slick-dots w-2/6 md:w-1/4 mx-auto">
               {!nowcastError && (
-                <SuctionTensionViz
-                  depth30Level={
+                <GroundLayersViz
+                  depth30StatusId={
                     nowcastData && nowcastData[0].value
-                      ? mapSuctionTensionToLevel(nowcastData[0].value)
+                      ? mapSuctionTensionToStatus(nowcastData[0].value)?.id
                       : undefined
                   }
-                  depth60Level={
+                  depth60StatusId={
                     nowcastData && nowcastData[1].value
-                      ? mapSuctionTensionToLevel(nowcastData[1].value)
+                      ? mapSuctionTensionToStatus(nowcastData[1].value)?.id
                       : undefined
                   }
-                  depth90Level={
+                  depth90StatusId={
                     nowcastData && nowcastData[2].value
-                      ? mapSuctionTensionToLevel(nowcastData[2].value)
+                      ? mapSuctionTensionToStatus(nowcastData[2].value)?.id
                       : undefined
                   }
-                  averageLevel={avgLevel}
+                  averageStatusId={avgLevel}
                 />
               )}
               {!forecastError && forecastData && forecastData?.length > 0 && (
-                <div className="bg-scale-4 h-full">
-                  <code>
-                    <pre>{JSON.stringify(forecastData, null, 2)}</pre>
-                  </code>
-                </div>
+                <ForecastViz
+                  // TODO: Attention: this is sample data. Replace with actual data once we have access:
+                  data={Array.from(Array(14)).map((_, i: number) => {
+                    return {
+                      date: addDays(Date.now(), i),
+                      waterSupplyStatusId: ['good', 'medium', 'critical'][
+                        Math.floor(Math.random() * 3)
```

Comment:
We should replace that with real data. Normally we should become this data via the `useForecastData` hook right?

Reply:
True, thanks for the reminder. It's just that we don't have any forecast data yet, so there won't be any viz to display. Would you say to already use the real data anyway?

Reply:
I did it! :)

Reply:
Nice, thank you!